[U] Update entry for Weideriche_ (#172)
one-among-us/data: people/Weideriche_/page.en.md

Zihe was a non-binary ally of us, whose pronouns were always She/Her.

> In a word, she was a girl who was gentle and broad-minded, but she couldn't get enough happiness and love.

Reviewer: ```suggestion
Generally speaking, she was a gentle and broad-minded girl, but she couldn't get enough happiness and love.
```
